Validation_Tests/ValidationsTests/IsEmptyValidationTests/IsEmptyValidationTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using FluentAssertions;
+using FluentAssertions.Execution;
+using PopValidations.Validations;
+using System.Collections.Generic;
+using Xunit;
+
+namespace PopValidations_Tests.ValidationsTests;
+
+public class IsEmptyValidationTests
+{
+    [Fact]
+    public void WhenSupplyingANullValue_ItValidatesAsSuccessful()
+    {
+        // Arrange
+        var validator = new IsEmptyValidation();
+
+        // Act
+        var result = validator.Validate(null);

```

Comment:
You might want to consider spliting this into 2 tests.
and using a theory.
Where test 1)  runs through all null theory data, and proves "Is Null or empty"
Where test 2) runs through all non-null theory data, and proves its "Not Null"

Yes this is similar to the "Integration" test you did with the runner.
That's the proof, that the system does what your expecting.
This should prove that the validation does what your expecting, and be more explicit, detailed, etc.